src/taipy/templates/multi-page-gui-core/{{cookiecutter.application_name}}/pages/data/data.md

```diff
@@ -0,0 +1,11 @@
+# Data
+
+Choose the CSV file: <|{input_csv_file}|file_selector|active={scenario}|label=Upload CSV|on_action=drop_csv|extensions=.csv|>
+
+<|{inputs}|table|rebuild|>
+
+Replacement value: <|{replacement_type}|input|active={scenario}|>
+
+<|Clean data|button|active={input_csv_file}|on_action=on_clean_data|>
+
+<|{results}|table|rebuild|>
```

Comment:
It's not generic enough. I am going to propose a generic page design to implement.